BLT/BLT/Focus Mode View/FocusTimer.swift

```diff
@@ -19,6 +19,8 @@ class FocusTimer {
 	var secs: Int {
 		return Int(cdt) % 60
 	}
+    ///amount of time that the timer has been running for
+    var timeStudied: TimeInterval
```

Comment:
This is a very inefficient way of doing this.

Reply:
This also does not fit with the description of the PR at all.

Reply:
Yeah I added some extra stuff and forgot about it. The time studied is simply the time studied during a single focus session. That then gets added to the variable focusTime in userData